### DISCUSSION TITLE
Zellij Running in a Web Browser

### DISCUSSION THREAD

**Discussion by nlvw** [zellij-org/zellij#1498] [Show and tell]
<img width="856" alt="image" src="https://user-images.githubusercontent.com/5723368/173659107-4e76910d-f6af-4bac-a7b1-d7e26d07810b.png">

Hi All,

I just wanted to show off Zellij running inside a web browser on a Supercomputer (HPC Cluster).  With the simplified UI and [https://github.com/tsl0922/ttyd](https://github.com/tsl0922/ttyd) providing the web server it looks really good.

A few things I haven't solved are:
1. Getting the clipboard to work without the shift key override. 
2. 'Ctrl + t' opens a new chrome tab and Zellij never sees the input (a lot of Ctrl shortcuts have this issue).  
3. Zooming in and out causes rendering issues so it's best to set the font size in ttyd before hand and not to try to resize on the fly.

Hopefully the planned web server feature solves a lot of this.  In the meantime let me know if you want me to test something related.

**Comment by a-kenji** [comment]
This is very awesome, thank you for sharing!
Please don't hesitate reaching out in the future.


You might be able to configure the copy command itself:
```
zellij setup --dump-config | rg copy
```

**Comment by lfmunoz** [comment]
@nlvw do you have instructions on how to get this to work on my web browser?
